docs/azure/README.md

```diff
@@ -0,0 +1,59 @@
+# Running a node on Microsoft Azure
+
+TL;DR copy/paste:
+
+```bash
+docker login azure
+docker context create aci starknet-aci
+docker context use starknet-aci
+docker volume create goerli-data --storage-account starknetnodes
```

Comment:
Where exactly is this `starknetnodes` storage account created?

Reply:
it is created if it does not exists, see [aci integration](https://docs.docker.com/cloud/aci-integration/#:~:text=By%20default%2C%20if%20the%20storage%20account%20does%20not%20already%20exist%2C%20this%20command%20creates%20a%20new%20storage%20account)